tests/command_line/test_dlsnxs2cbf.py

```diff
@@ -6,7 +6,7 @@
 
 
 def test_dlsnxs2cbf(dials_data, tmpdir):
-    screen = dials_data("i04_eiger_screen")
+    screen = dials_data("thaumatin_eiger_screen")
     master = os.path.join(screen, "Therm_6_1_master.h5")
```

Comment:
`dials_data()` returns a py.path object (same as `tmpdir`), so you can't use `os.path` methods on it.
You can do
```python
master = screen.join("Therm_6_1_master.h5")
```
procrunner understands py.path objects.